tests/blackbox_test.go

```diff
@@ -311,3 +359,32 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 	}
 	return nil
 }
+
+// Identify and replace $uuid<num> with correct UUID
+// Variables with matching <num> should have identical UUIDs
+func replaceUUIDVars(t *testing.T, str string, UUIDmap map[int]string) (string, error) {
+	finalStr := str
+
+	pattern := regexp.MustCompile("\\$uuid([0-9]+)")
+	for _, match := range pattern.FindAllStringSubmatch(str, -1) {
+		if len(match) != 2 {
+			return str, fmt.Errorf("find all string submatch error: want length of 2, got length of %d", len(match))
+		}
+
+		num_int, err := strconv.Atoi(match[1])
+		if err != nil {
+			return str, fmt.Errorf("atoi failed: %v\n%s", err, num_int)
```

Comment:
should be %d or %v. Travis is failing because of this

Reply:
Updated.